AES.ApplicationSvc/ApplicationSvc.svc.cs

```diff
@@ -133,19 +167,66 @@ public ApplicationInfoContract GetApplication(ApplicantInfoContract user)
             return retApp;
         }
 
-        public ApplicationInfoContract GetCallApplication(UserInfoContract user)
+        public ApplicationInfoContract GetCallApplication(ApplicantInfoContract user)
         {
-            throw new NotImplementedException();
+            return GetApplication(user);
         }
 
         public ApplicationInfoContract GetInterviewApplication(UserInfoContract user)
         {
             throw new NotImplementedException();
         }
 
-        public bool PullApplicantFromCallQueue(UserInfoContract user)
+        private bool SetApplicationStatus(int applicantID, AppStatus expectedStatus, AppStatus setStatus)
```

Comment:
Keep all the private stuff at the bottom, and API stuff at the top. Preferably organized alphabetically in each section.